cmd/server_test.go

```diff
@@ -57,7 +57,7 @@ var testFlags = map[string]interface{}{
 	ADWebhookPasswordFlag:            "ad-wh-pass",
 	ADWebhookUserFlag:                "ad-wh-user",
 	AtlantisURLFlag:                  "url",
-	AllowCommandsFlag:                "version,plan,unlock,import,approve_policies", // apply is disabled by DisableApply
+	AllowCommandsFlag:                "version,plan,unlock,import,approve_policies", // apply is disabled by disableApply
```

Comment:
can we keep this all lowercase for now?

Reply:
@nitrocode reverted these changes. now ready for review 🙇